config/crds/hive_v1_clusterdeployment.yaml

```diff
@@ -2,15 +2,13 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
   name: clusterdeployments.hive.openshift.io
 spec:
   additionalPrinterColumns:
   - JSONPath: .spec.clusterName
     name: ClusterName
     type: string
-  - JSONPath: .metadata.labels.hive\.openshift\.io/cluster-type
+  - JSONPath: .metadata.labels[hive.openshift.io/cluster-type]
```

Comment:
The failure is this change in lines right here, the new syntax appears to break the additional printer columns, at least on the versions of kube we're running against. The result is no additional printer columns, which fails our test looking for BASEDOMAIN.

Derek mentioned this column was a weird thing to display awhile back and I'd been meaning to remove it since. Rather than fight this I'm going to just take it out.

Reply:
Is there any other user that is relying on that column? Would it be easier to keep it in as the older version for now?

Checkout the master version of the file and update the CRDs from that version.

Reply:
I have doubts anyone is using it and I can't see where this would fit into any AppSRE/OCM/SRE-P scripting, but I don't think we should be considering additional printer columns to be an API. If this does happen to break some pipeline or automation, that should be addressed anyhow.

Reply:
Here is an example of a place where additional printer columns are used by users. I am fine if we want to make this change, but I would rather make it as a separate PR rather than burying it in a larger PR.

https://github.com/openshift/managed-cluster-config/blob/0f29251676f95c9414c9326cc43afe652246cc0d/scripts/rotate-aws-creds.sh#L93

Reply:
I don't think that is using any of our additional printer columns. 

I will go get the master file copy and hand edit it for the new field and try to get this all updated properly, then submit a new PR.

Reply:
You should not need to hand-edit anything.
```
git checkout master -- config/crds
make update-codegen-crds
```

Reply:
This picks up an unrelated change that removes syncsetinstance Applied printer column for some reason. I'll just got with the hand edit.